server/utils.go

```diff
@@ -295,3 +296,7 @@ func getSourceMount(source string, mountinfos []*mount.Info) (path, optional str
 
 	return res.Mountpoint, res.Optional, nil
 }
+
+func isContextError(err error) bool {
+	return err == context.Canceled || err == context.DeadlineExceeded
```

Comment:
Have we identified cases in which we want to retain a resource when the context is Canceled from the kubelet?

Reply:
I figured we'd keep the behavior as we had before--checking both Canceled and DeadlineExceeded